docs/reference/eql/eql-search-api.asciidoc

```diff
@@ -89,6 +89,17 @@ request that targets only `bar*` still returns an error.
 +
 Defaults to `true`.
 
+`ccs_minimize_roundtrips`::
+(Optional, Boolean)
++
+If `true`, network round-trips between the local and the remote cluster are
+minimized when executing cross-cluster search (CCS) requests.
++
+This setting is currently only effective if the request does not include both
```

Comment:
I'd use `option` or `parameter` to avoid confusion with cluster/index settings.
```suggestion
This option is currently only effective if the request does not include both
```